### DISCUSSION TITLE
Activate indent-blankline in .vimrc

### DISCUSSION THREAD

**Discussion by ivan94fi** [lukas-reineke/indent-blankline.nvim#391] [Q&A]
Hi, thank you for the awesome plugin!

I am having troubles with the configuration and I figured this was the right place to ask. I installed indent-blankline with vim-plug in NVIM v0.5.1, but the indentation guides are not active unless I manually activate the plugin with `:IndentBlanklineEnable!`.

I am using a .vimrc for configuration (not a lua file) so I tried calling the setup function in my .vimrc with something like
```vim
lua require("indent_blankline").setup {}
``` 
or
```vim
lua << EOF
require("indent_blankline").setup {
    show_current_context = true,
    show_current_context_start = true,
}
EOF
```

These however did not work, so I ended up doing this:
```vim
autocmd BufEnter * execute ':IndentBlanklineEnable'
```
which works as expected, but I feel is not really the right way to go.

So my question is: what is the right way of activating the plugin in a .vimrc file?

**Comment by lukas-reineke** [comment]
You don't have to do any of that, it should just work with no configuration.

Calling `IndentBlanklineEnable` overwrites all disable checks, that is probably not what you want.
Are you sure there is no other configuration for indent-blankline somewhere in your config?

**Reply by ivan94fi** [reply]
Ok I was pretty sure there was something not right with my method.


So I tried using this as vim config:
```vim
filetype plugin on

" Plugins
call plug#begin('~/.vim/plugged')

Plug 'lukas-reineke/indent-blankline.nvim'

call plug#end()
```
and opening this python file
```python
import random

def f():
    if random.random() < 0.5:
        return random.randint(0, 234523452345)
```
with this command `vim -u debug_vimrc file.py`, but the indent lines are not showing. If I manually execute `:IndentBlanklineEnable`, the indent lines show up fine.

What could be wrong?

**Reply by lukas-reineke** [reply]
There was a small bug that broke the enable check for Neovim 0.5.1.
I released a fix, it should work now.

But I recommend updating, 0.5.1 is pretty out of date. I try to keep the code backwards compatible, but it is not always possible. You will have problems with other plugins as well.

**Reply by ivan94fi** [reply]
Yes, I can confirm that the fix is working, thank you.

> But I recommend updating, 0.5.1 is pretty out of date

Well, in NeoVim releases page 0.5.1 is marked as released on 27 September 2021, so it is not really really old, but I guess you could call it pretty out of date. Anyway I will follow your suggestion and update as soon as I can. Thanks again.
